additional/docs/blog/posts/vcmi-1.5.0-release.md

```diff
@@ -14,16 +14,34 @@ A new major release has been published. VCMI 1.5.0 sees the light of day. The ma
 ## Online multiplayer support
 VCMI now has an online lobby that can be used to set up multiplayer sessions. But you can also interact with other players. The lobby works on all platforms and regardless of firewall restrictions. It can be opened at any time on desktop systems with `CTRL + Tab` and on mobile platforms with a three-finger touch.
```

Comment:
I would expand this and add something like "Additionally it is now possible to check towns/heroes during opponent player turn and bypass hero movement latency for non-host players by picking maximum hero speed in game options."